benchmarks/trees/score.py

```diff
@@ -207,7 +208,10 @@ def predict(self, predict_data):
                 end = min(start + batch_size, total_size)
 
                 if self.params["operator"] == "xgb":
-                    self.predictions[start:end, :] = self.sess.run([output_name], {input_name: predict_data[start:end, :]})
+                    pred = self.sess.run([output_name], {input_name: predict_data[start:end, :]})
+                    if type(pred) is list:
```

Comment:
is `pred` ever an empty list?  (like if error?)

Reply:
If there is an error this should crash instead of returning an empty list. If the input is empty probably we will get an empty list tho, but this should not happen.